src/Servers/Kestrel/Core/src/Adapter/Internal/LoggingConnectionAdapter.cs

```diff
@@ -6,9 +6,9 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Core.Adapter.Internal
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Adapter
 {
-    public class LoggingConnectionAdapter : IConnectionAdapter
+    internal class LoggingConnectionAdapter : IConnectionAdapter
```

Comment:
Why are we bothering to change the namespaces? Shouldn't making the classes internal be enough?

Reply:
That's what @pranavkm did with his PRs. Keeping consistent.

Reply:
Did @pranavkm still keep the classes moved out of the Internal namespace in 
"Internal" directories? I think it's better to have the namespace structure match the directory structure if possible.

Reply:
I emptied .Internal folder along with changing the namespace. In MVC it helped do some code organization since we typically had one .Internal namespace per project which acted as a dumping ground for all kinds of features.

Reply:
I see. @halter73 I agree with you then with keeping the .Internal namespace as it isn't really a dumping ground for us.